2-mandatory/1-writers.js

```diff
@@ -72,6 +77,13 @@ Exercise 2:
 
 function logDeadWritersInTheirForties() {
   // write your code here
+  writers
+    .filter((person) => person.age >= 40 && person.age <= 49 && !person.alive)
```

Comment:
Well Done Meysam.
Nice use of .filter.
I was wondering if you can write this function with for loop.
And also break down the conditional operator and store each conditions in separate variable.

Great job 👍🏼